### PR TITLE
[OSDOCS-8987]: OSD-GCP - Support A2 instance type (with GPU)

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.15+
Issue:

https://issues.redhat.com/browse/OSDOCS-8987
Link to docs preview:

https://75453--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/osd-service-definition.html#gcp-compute-types_osd-service-definition
QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mletalie** [comment]
Feature is still in process, do not release docs until greenlit.

**Comment by mletalie** [comment]
Hello @lento-sun,
   Can you please review this PR when you get a moment. Feature still being worked on, but I believe there will not be additional changes prior to release. Thanks!

**Comment by mletalie** [comment]
Based on today's conversation in XCMSTRAT-175 sync up, it was determined that 8g and 16g instance types should not be included in this PR. Removed those instances until further notice.

File: modules/sdpolicy-am-gcp-compute-types.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {product-title} offers the following worker node types and sizes on Google Cloud that are chosen to have a common CPU and memory capacity that are the same as other cloud instance types:
 [NOTE]
 ====
-`e2`, `n2` and `c2` compute types are available for CCS only.
+`e2`, `n2`, `c2` and `a2` compute types are available for CCS only.
```
Comment (jneczypor) [inline modules/sdpolicy-am-gcp-compute-types.adoc:11]:
```suggestion
`e2`, `n2`, `c2`, and `a2` compute types are available for CCS only.
```
[IBM style guide](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series) uses the Oxford comma (the comma before the conjunction).

**Review by jneczypor — COMMENTED** [review]
Just one small change. I'm still onboarding, so @michaelryanpeter will follow after with his confirmation. Thanks!

**Comment by michaelryanpeter** [comment]
+1 to @jneczypor's comment
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**